### PR TITLE
Sort dicts by key in force_string

### LINKED ISSUE #58: Tasks with autoretry_for stuck in locked state
I've discovered a bug in `get_key` method which prevents lock release.
```
@app.task(name='mytask', base=celery_once.QueueOnce)
def mytask(foo, bar):
    ...

mytask.delay(foo=1, bar=2)
```
The `getcallargs(self.run, *args, **kwargs)` call returns `{'foo': 1, 'bar': 2}`. 
The `get_key` call returns `qo_mytask_foo-1_bar-2`. Everything works properly.

If I'd add `autoretry_for` argument:
```
task(name='mytask_retry', base=celery_once.QueueOnce, autoretry_for=(Exception, ))
def mytask_retry(foo, bar):
    ...

mytask_retry.delay(foo=1, bar=2)
```

The `getcallargs(self.run, *args, **kwargs)` call returns `{'args': (), 'kwargs': {'foo': 1, 'bar': 2}}`. 
The `get_key` call returns `qo_mytask_retry_args-()_kwargs={'bar': 2, 'foo': 1}`. 
The `{'bar': 2, 'foo': 1}` part is unsorted and sometimes the order changes to  `{'foo': 1, 'bar': 2}`. 

The same issue takes place when a nested dict is passed as task's argument. For example, `mytask(foo={'spam': 10, 'eggs': 20}, bar=2)`. The `get_key` call returns `qo_mytask_foo-{'eggs': 20, 'spam': 10}_bar-2` and sometimes `qo_mytask_foo-{'spam': 10, 'eggs': 20}_bar-2`

This lead to lock release issues because `get_key` result is different on lock stage and release stage.

### PR DESCRIPTION
**Pull request by andbortnik**
This closes #58 issue

### CODE REVIEW

**Comment by cameronmaske** [comment]
Hey andbortnik, great find and thanks for the patch (sorry for the delayed reply). 

The one suggestion I'd like to implement, in order to make sure the keys as compatible backwards would be to use [`OrderedDict `](https://docs.python.org/2/library/collections.html#collections.OrderedDict) instead of Tuples. 

This will ensure that if any existing keys (and therefore locks) that are in redis, upgrading won't lose the locks (unless the bug you noted occurs). 

Would you be comfortable making those changes? If not, I will try to get to them when I'm not swamped with work.

File: celery_once/helpers.py [hunk 1]
```diff
@@ -27,14 +28,20 @@ def now_unix():
...
 def force_string(kwargs):
     """
     Force key in dict or list to a string.
     Fixes: https://github.com/TrackMaven/celery-once/issues/11
     """
     if isinstance(kwargs, dict):
-        return {
-            force_string(key): force_string(value) for key, value in six.iteritems(kwargs)}
+        return [
+            (force_string(key), force_string(value))
```
Comment (cameronmaske) [inline celery_once/helpers.py]: Would change this to use OrderedDict (ref: https://stackoverflow.com/questions/9001509/how-can-i-sort-a-dictionary-by-key)

**Comment by andbortnik** [comment]
Hey @cameronmaske, thank you, the comment make sense. I'll try to fix it this week as soon as I have time.

**Comment by cameronmaske** [comment]
@andbortnik Thanks for this PR, it's now released as `1.3.0`.
Sorry again for the delay getting around to it!

**Pull request closed without merging**